concrete/single_pages/dashboard/system/registration/deactivation.php

```diff
@@ -1,88 +1,78 @@
-                    </div>
-                </div>
-            </fieldset>
-
-            <fieldset>
-                <legend><?=t('Automatic User Deactivation')?></legend>
-                <div class="form-group">
-                    <div class="checkbox">
-                        <label>
-                            <?=$form->checkbox('enableAutomaticUserDeactivation', 1, $enableAutomaticUserDeactivation)?>
-                            <?=t('Automatically deactivate users when they have not logged in for awhile. Users will need to be manually reactivated.')?>
-                        </label>
-                    </div>
-                </div>
-                <div class="well" data-group="user-deactivation-days">
-                    <div class="form-group">
-                        <label class="control-label"><?=t('Threshold')?></label>
-                        <div class="form-inline">
-                            <p><?=t('Deactivate users when they have not logged in for %s days',
-                                $form->number('userDeactivationDays', $userDeactivationDays, ['style' => 'width: 100px'])
-                            )?></p>
-                        </div>
-                    </div>
-
-                    <div class="help-block">
-                        <?=t('Note: You will need to run the "Deactivate Users" job in order for these deactivations to actually occur.')?>
-                    </div>
-                </div>
-
-                <div class="form-group">
-                    <div class="checkbox">
-                        <label>
-                            <?=$form->checkbox('enableLogoutDeactivation', 1, $enableLogoutDeactivation)?>
-                            <?=t('Automatically deactivate users after failed login attempts.')?>
-                        </label>
-                    </div>
-                </div>
-                <div class="well" data-group="user-deactivation-logins">
-                    <div class="form-group" data-group="user-deactivation-logins">
-                        <label class="control-label"><?=t('Threshold')?></label>
-                        <div class="form-inline">
-                            <p>
-                                <?= t('Deactivate users once they have failed %s login(s) within %s seconds',
-                                    $form->number('userLoginAmount', $userLoginAmount, ['style' => 'width: 100px']),
-                                    $form->number('userLoginDuration', $userLoginDuration, ['style' => 'width: 100px'])
-                                ) ?>
-                            </p>
+    <div class="form-group">
+        <?= $form->label('inactiveMessage', t('Inactive User Error Message')) ?>
+        <?= $form->textarea('inactiveMessage', $inactiveMessage, ['rows' => 4, 'cols' => 10]) ?>
+        <small class="text-mited"><?= t('This message will be shown to inactive users when they attempt to login.') ?></small>
```

Comment:
We should fix this post-merge, I t think this is the wrong class.

Reply:
See https://github.com/concrete5/concrete5/pull/8693